Configuration/Eras/python/Modifier_run3_muon_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+run3_muon=cms.Modifier()
```

Comment:
is it still needed in the current implementation ? I don't think so.

Reply:
I think for [this comment](https://github.com/cms-sw/cmssw/pull/44322#issuecomment-1994508415), run3_muon is actually useful. So I'd be inclined to keep it.